docs/README.md

```diff
@@ -0,0 +1,200 @@
+# Map of pages
+
+This document is create due to better orientation in the page components. 
+
+
+this documents speaks about:
+- 1. general architecture of our classes 
+- 2. identification of used elements
+
+## 1. General architecture
+
+Generally 'Page' or 'Component' suffix of java class provides information,
+that such class represents web element, defined in the Element.ROOT variable.
+If there are any sub-elements these are listed in private static classes, like 
+Input, Select, Button, or generally: Element
+Disadvantage of this approach is that such 'Pages' does not represent web 'pages' as user sees them.
+
+To partly remedy this disadvantage class 'SyndesisPage' has been created.
+This class represents web 'page' from user point of view.
```

Comment:
Only small remark. 
I'm not sure, but I think this could be little bit confusing in context of previous sentences. SyndesisPage class is only enumeration, which give you opportunity to get instance of page object (even so called "component") based on defined natural language alias, nothing more.